06_puppy/sirigithub/mapStore.go

```diff
@@ -0,0 +1,52 @@
+package main
```

Comment:
filenames are typically all lower case in go (below as well).

Reply:
Noted. Done.